samples/nrf_compress/mcuboot_update/README.rst

```diff
@@ -0,0 +1,78 @@
+.. _mcuboot_compressed_update:
+
+MCUboot Compressed Update
+#########################
```

Comment:
To solve the doc build, this file's path should be added to https://github.com/nrfconnect/sdk-nrf/blob/main/doc/nrf/samples/other.rst (or other section on the same level, but I guess this one is the best), so that the readme is included in the doc structure.